### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by stevenzeiler**

### CODE REVIEW

**Comment by stevenzeiler** [comment]
+r

File: index.html [hunk 1]
```diff
@@ -17,6 +17,8 @@
   <link rel="apple-touch-icon-precomposed" sizes="72x72" href="/apple-touch-icon-72x72-precomposed.png">
   <link rel="apple-touch-icon-precomposed" href="/apple-touch-fa-57x57-precomposed.png">
 
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
```
Comment (tdfischer) [inline index.html:20]: 300italic? Typo maybe? Not actually familiar with the google fonts api.

File: index.html [hunk 2]
```diff
@@ -35,7 +37,8 @@
     }
 
     thead {
-      background: #C3D9FF none repeat scroll 0% 0%;
+      /*background: #eaf0f7 none repeat scroll 0% 0%;*/
```
Comment (tdfischer) [inline index.html]: Commented code should just be dropped outright.

File: index.html (hunk 1 above)
Comment (stevenzeiler) [inline index.html:20]: Ben said to paste that in as is.

File: index.html [hunk 3]
```diff
@@ -119,25 +126,96 @@
...
+        -webkit-transition: .5s;
+        transition: .5s;
+      }
+    </style>
+    <style>
+      /* colors */
+      h1, h2, h3, h4, h5, h6, a {
+        color: #346AA9; /* @blue */
+      }
+/*@darkblue:
```
Comment (tdfischer) [inline index.html]: more dead CSS

File: index.html (hunk 2 above)
Comment (stevenzeiler) [inline index.html]: +1

File: package.json [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
+  "version": "0.1.1",
   "dependencies": {
     "ejs": "^2.3.2",
     "express": "^4.12.4"
   },
   "repository": {},
   "scripts": {
-    "start": "node server.js"
+    "start": "node server.js",
+    "test": "echo 'insert full test cover here'"
```
Comment (tdfischer) [inline package.json:11]: :+1:

**Comment by tdfischer** [comment]
A few minor issues about removing dead code, otherwise LGTM :+1:

**Pull request merged**